.env.example

```diff
@@ -80,18 +80,16 @@ SUBDOMAIN_APIDOCS=apidocs.
 SUBDOMAIN_REDISADMIN=redis.
 
 # Paths in the filesystem
-PATH_OMS_GLOBAL=oms-global/docker/
+PATH_GATEWAYS=gateways/docker/
 PATH_CORE=core/docker/
 PATH_EVENTS=events/docker/
 PATH_STATUTORY=statutory/docker
 PATH_FRONTEND=frontend/docker/
 PATH_DISCOUNTS=discounts/docker/
 PATH_MAILER=mailer/docker/
-PATH_OMS_SECURITY=oms-security/docker/
-PATH_OMS_SHORTENER=oms-shortener/docker/
-PATH_OMS_MONITOR=oms-monitor/docker/
+PATH_PASS_MANAGER=pass-manager/docker/
+PATH_MONITOR=monitor/docker/
 PATH_OMS_ELASTIC=oms-elastic/docker/
```

Comment:
good catch, I did not truly forget though, it will be part of another PR where we remove dead code (in this case, elastic is no more self-hosted but provided as donation, so I will remove the reference)